src/formatter/formatComplexDataStructure.js

```diff
@@ -0,0 +1,67 @@
+/* @flow */
+
+import collapse from 'collapse-white-space';
```

Comment:
What do you mean by "complexDataStructure"? Seems to be objects or arrays that could also be